java_hotspot_grpc_g1gc_bench/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk:16_36-jdk-hotspot
+FROM openjdk:17
```

Comment:
Does the name "hotspot" still make sense here for those java benchmarks? Or should we remove this from their names?